modules/ols-about-azure-openai.adoc

```diff
@@ -5,6 +5,6 @@
 [id="ols-about-azure-openai_{context}"]
 = About Azure OpenAI 
 
-To use {azure-openai} with {ols-full}, you must have access to {azure-openai}. If you have already applied for and received access to {azure-openai}, you can ignore the rest of the information in this topic.
+To use {azure-official} with {ols-full}, you must have access to {azure-openai}. If you have already applied for and received access to {azure-openai}, you can ignore the rest of the information in this topic.
 
-{azure-openai} access currently requires an approval process initiated from within the {azure-full} portal. It can take several days for Microsoft to approve your access. When your access to {azure-openai} is approved, make sure you know which Azure subscription your {azure-openai} access is associated with.
+{azure-official} access requires an approval process initiated from within the {azure-official} portal. It can take several days to obtain approval for access. When your access to {azure-official} is approved, make sure you know which {azure-official} subscription your {azure-openai} access is associated with.
```

Comment:
this is actually no longer required and can be removed.

Reply:
done